ouroboros-consensus-cardano/src/ouroboros-consensus-cardano/Ouroboros/Consensus/Cardano/Node.hs

```diff
@@ -522,7 +522,7 @@ pattern CardanoNodeToClientVersion12 =
       :* EraNodeToClientEnabled ShelleyNodeToClientVersion7
       :* EraNodeToClientEnabled ShelleyNodeToClientVersion7
       :* EraNodeToClientEnabled ShelleyNodeToClientVersion7
-      :* EraNodeToClientEnabled ShelleyNodeToClientVersion7
+      :* EraNodeToClientEnabled ShelleyNodeToClientVersion8
```

Comment:
> Would it just return Nothing when the inquired ledger state is not yet a Conway ledger state?

Yes, but only for now. As in, this is functionality currently part of the `EraGovernance` class for now, but will soon move to another  new `ConwayEraGovernance` class, which will be applicable to eras Conway onwards. So this functionality is currently available and will return `Nothing` pre-conway. But going forward, it wont be available pre-conway, as the older eras wont implement instances of the new class. (Please correct me if I'm wrong @lehins @Soupstraw)

>  but we can also keep it as is.

If it can be kept as available only from Conway onwards in this instance without causing trouble to older eras, that would be more convenient I suppose.